internal/pkg/print/print.go

```diff
@@ -15,8 +15,10 @@ const (
 	WithAggregateTypeDefaults
 	// WithRequired prints if inputs are required
 	WithRequired
-	// WithSorting prints sorted inputs and outputs
-	WithSorting
+	// WithSortInputsByRequired prints inputs with required fields first and then by name
```

Comment:
Ask asked before, please change this description to something that's better understandable. Preferrably "WithSortInputsByRequired sorts inputs by name and prints required inputs first". Please also use this where you define `--sort-inputs-by-required` in `main.tf`s and in the documentation in `main.tf` and `README.md`.